### ISSUE TITLE
Bogus 'flat' has been replaced with 'variant="flat"' for VBtn

### ISSUE THREAD

**Issue by Glandos** [vuetifyjs/eslint-plugin-vuetify#131]
I'm using `vuetify.configs['flat/recommended-v4']`, and the following template reports an error:
```html
                        <v-btn
                          size="small"
                          flat
                          density="comfortable"
                          :icon="mdiContentCopy"
                          @click="copyContentToClipboard"
                        />
```

However, [documentation mentions](https://vuetifyjs.com/en/api/v-btn/#props-flat):
```
flat

boolean	

false

Removes the button box shadow. This is different than using the ‘flat’ variant.
```

So, to me, this should be excluded from rule here. Or documentation should be updated?
